Libraries/LibWeb/Parser/HTMLTokenizer.cpp

```diff
@@ -1038,6 +1042,20 @@ Optional<HTMLToken> HTMLTokenizer::next_token()
                 {
                     SWITCH_TO(BeforeAttributeValue);
                 }
+                ON_ASCII_UPPER_ALPHA
+                {
+                    m_current_token.m_tag.attributes.last().local_name_builder.append_codepoint(tolower(current_input_character.value()));
+                    continue;
+                }
+                ON(0)
+                {
+                    PARSE_ERROR();
+                    m_current_token.m_tag.attributes.last().local_name_builder.append_codepoint(0xFFFD);
+                    continue;
+                }
+                ON('"') { PARSE_ERROR(); }
+                ON('\'') { PARSE_ERROR(); }
+                ON('<') { PARSE_ERROR(); }
```

Comment:
These look diffferent from the rest of the file.
Also `PARSE_ERROR` does not implicitly return. If we want to skip ahead to the "anything else" case, let's make that explicit with a `goto` :)